zqd/storage/archive/archive.go

```diff
@@ -0,0 +1,81 @@
+package archivestore
```

Comment:
The mismatch between directory and package name here is surprising. Why not change the directory name to archivestore?